### PR TITLE
chore(dashboards): add is_hidden column to dashboardwidgetquery table

### PR DESCRIPTION
**Pull request by obostjancic**
Adds  is_hidden column to dashboardwidgetquery table

- Required for #67328

### CODE REVIEW

File: src/sentry/models/dashboard_widget.py [hunk 1]
```diff
@@ -133,6 +133,8 @@ class DashboardWidgetQuery(Model):
     order = BoundedPositiveIntegerField()
     date_added = models.DateTimeField(default=timezone.now)
     date_modified = models.DateTimeField(default=timezone.now)
+    # Whether this query is hidden from the UI, used by metric widgets
+    is_hidden = models.BooleanField(null=True)
```
Comment (wedamija) [inline src/sentry/models/dashboard_widget.py:137]: Does it make sense to make this not null and default to False? https://develop.sentry.dev/database-migrations/#adding-columns-with-a-default

**Review by wedamija — APPROVED** [review]
Migration lgtm

File: src/sentry/models/dashboard_widget.py (hunk 1 above)
Comment (obostjancic) [reply src/sentry/models/dashboard_widget.py]: It does! Thanks for pointing it out! I updated the PR, would you mind taking another quick look? thanks!

**Review by markstory — APPROVED** [review]
Schema looks good.

**Review by wedamija — APPROVED** [review]

**Comment by obostjancic** [comment]
Had a conflict with another migration, rebased master and regenerated this migration again

**Pull request merged**